R/utils-plot.R

```diff
@@ -105,11 +105,13 @@ extendYaxisLimits <- function(gsplot, error_bar_args){
   side <- ifelse(!is.null(error_bar_args[['side']]), error_bar_args[['side']], 2)
   side_nm <- paste0('side.', side)
   
+  ##Find the max and min of errors with bars.
   lowest_error_bar <- min(error_bar_args[['y']] - error_bar_args[['y.low']])
-  lowest_y <- min(ylim(gsplot, side=side)[1], lowest_error_bar)
-  
   highest_error_bar <- max(error_bar_args[['y']] + error_bar_args[['y.high']])
-  highest_y <- max(ylim(gsplot, side=side)[2], highest_error_bar)
+  
+  ##Compare error bar extrema with current plot lims.
+  lowest_y <- min( min(ylim(gsplot, side=side)), lowest_error_bar)
+  highest_y <- max( max(ylim(gsplot, side=side)), highest_error_bar)
```

Comment:
if the axis is reversed, wouldn't lowest_y be wrong now? I think that's why we were using `ylim(...)[1]`